scripts/generate_curriculum_from_images.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+from pathlib import Path
+
+from PIL.Image import Resampling
+from yaml import load, dump
+
+IMAGE_SIZE = (640, 360)
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+from PIL import Image
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        "Create new curriculum from directory full of objects. Take language "
+        "descriptions from filenames. "
+    )
+    parser.add_argument("images_dir", type=Path)
+    parser.add_argument(
+        "--output_dir", type=Path, default="data/multi_obj")
+    args = parser.parse_args()
+
+    args.output_dir.mkdir(parents=True)
+
+    situations_count = 0
+    images_dir: Path = args.images_dir
+    for image_file in images_dir.glob('*.jpg'):
+        input_image = Image.open(image_file)
+        description = image_file.name[:-len('.jpg')]
```

Comment:
You can use `image_file.stem` here (if I recall the attribute correctly) to get the path's base name without the extension/suffix.